setup.cfg

```diff
@@ -1,15 +1,42 @@
 [metadata]
-name=stsci.imagestats
-description=Compute sigma-clipped statistics on data arrays
-long_description=README.rst
-long_description_content_type=text/x-rst
-author=Warren Hack, Christopher Hanley
-author_email=help@stsci.edu
-license=BSD-3-Clause
-version=1.6.2
-version_date=28-March-2019
-edit_on_github=False
-url=http://www.stsci.edu/
+name = stsci.imagestats
+description = Compute sigma-clipped statistics on data arrays
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Warren Hack, Christopher Hanley
+author_email = help@stsci.edu
+license = BSD-3-Clause
+url = https://github.com/spacetelescope/stsci.imagestats
+project_urls =
+    Tracker = https://github.com/spacetelescope/stsci.imagestats/issues
+    Documentation = https://stsciimagestats.readthedocs.io/en/stable/
+    Source Code = https://github.com/spacetelescope/stsci.imagestats
+classifiers=
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Topic :: Scientific/Engineering :: Astronomy
+    Topic :: Software Development :: Libraries :: Python Modules
+    Development Status :: 7 - Inactive
+
+[options]
+zip_safe = False
+python_requires = >=3.6
+setup_requires =
+    setuptools_scm
+packages = find:
+namespace_packages =
+    stsci
+
+[options.extras_require]
+docs =
+    numpy
+    numpydoc
+    sphinx
+    sphinx-automodapi
+    sphinx-rtd-theme
+    stsci-rtd-theme
```

Comment:
Why two different themes?

Reply:
Great question!  It is what was there before.  I have no idea.  🤷

Reply:
Ah, it turns out `stsci-rtd-theme` depends on `sphinx-rtd-theme` being installed.  This is true for other packages that use it.